### PR TITLE
*: fix filter parameters with comma

### LINKED ISSUE #1783: Api v2: comma no longer allowed in get parameter
**What did you do?**

- Run an empty alertmanager
- Open http://127.0.0.1:9093/api/v2/alerts?silenced=true&inhibited=false&filter=recipient%3Dops%2Fticket%2Cmgmt%2Fsms

**What did you expect to see?**

The query to succeed

**What did you see instead? Under which circumstances?**

Error 400 + "bad matcher format: mgmt/sms"

**Environment**

* System information:

	2 different Linux distros

* Alertmanager version:

```
alertmanager, version 0.16.1 (branch: HEAD, revision: 571caec278be1f0dbadfdf5effd0bbea16562cfc)                                           
  build user:       root@3000aa3a06c5
  build date:       20190131-15:05:40
  go version:       go1.11.5
```

* Alertmanager configuration file:
default

### PR DESCRIPTION
**Pull request by simonpasquier**
Closes #1783 

cc @roidelapluie

### CODE REVIEW

**Comment by roidelapluie** [comment]
This is a good candidate for e2e/acceptance test. Do we have that kind of tests?

**Comment by roidelapluie** [comment]
Iirc maybe in the orchestrator tests. We can do it later.

File: ui/app/Makefile [hunk 1]
```diff
@@ -43,10 +43,7 @@ script.js: src/Data elm-env format $(ELM_FILES)
 
 src/Data: ../../api/v2/openapi.yaml
 	-rm -r src/Data
-	# TODO: Use released version instead of `@sha256:d9f87079bb4d7[...]` that
-	# includes patch for
-	# https://github.com/OpenAPITools/openapi-generator/issues/1465
-	docker run --user=$(shell id -u $(USER)):$(shell id -g $(USER)) --rm -v ${PWD}/../..:/local openapitools/openapi-generator-cli@sha256:d9f87079bb4d769e2c6f2a6b72677f4d5e64db687db0455eb41227663290e53c generate \
+	docker run --user=$(shell id -u $(USER)):$(shell id -g $(USER)) --rm -v ${PWD}/../..:/local openapitools/openapi-generator-cli:v3.3.4 generate \
```
Comment (stuartnelson3) [inline ui/app/Makefile]: 🥇

**Comment by mxinden** [comment]
@simonpasquier thanks for the patch.

> This is a good candidate for e2e/acceptance test. Do we have that kind of tests?

@roidelapluie would you like to write one? Should be fairly easy with the work from @stuartnelson3 here: https://github.com/prometheus/alertmanager/blob/master/test/with_api_v2/acceptance/api_test.go

**Comment by roidelapluie** [comment]
That work uses the GO directove directly ; we should use the http package instead. so it is definitively very complex.

**Pull request merged**